src/checkbox/src/Checkbox.js

```diff
@@ -73,6 +73,11 @@ class Checkbox extends PureComponent {
      */
     indeterminate: PropTypes.bool,
 
+    /**
+     * Function that returns the ref of the checkbox.
+     */
+    innerRef: PropTypes.func,
```

Comment:
Can you update the `index.d.ts` typedef too?

Reply:
👍